src/Microsoft.TestPlatform.Extensions.TrxLogger/ObjectModel/TestAggregation.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.Extensions.TrxLogger.ObjectModel
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.TestPlatform.Extensions.TrxLogger.XML;
+
+    /// <summary>
+    /// Test aggregation element.
+    /// </summary>
+    internal abstract class TestAggregation : TestElement, ITestAggregation
```

Comment:
>TestAggregation [](start = 28, length = 15)

can we rename to TestElementAggregation..?

Reply:
done.